api/envoy/admin/v3/config_dump.proto

```diff
@@ -17,6 +17,28 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
 // [#protodoc-title: ConfigDump]
 
+// Config status from the view of a xDS client, which tells the synchronization
+// status between the xDS client and the xDS server.
+enum ConfigErrorStatus {
+  // Config status is not available/unknown.
+  UNKNOWN = 0;
+
+  // Client requested the config but hasn't received any config from management
+  // server. The client will not fail traffic, but will queue them until update
+  // arrived or timed out.
```

Comment:
"arrives or the client times out waiting for the resource".

Reply:
Updated.